script.js

```diff
@@ -92,14 +81,42 @@ $('#pictureBook').on('change', function () {
 });
 
 $('#childrenBook').on('change', function () {
-    bookApp.radioButtonChangeHandler(childrenMiddleGrade)
+    const childrenBookRes = bookApp.radioButtonChangeHandler(childrenMiddleGrade)
 });
 
 $('#seriesBook').on('change', function () {
-    bookApp.radioButtonChangeHandler(seriesBooks)
+    const seriesBookRes = bookApp.radioButtonChangeHandler(seriesBooks)
 });
```

Comment:
Can you please elaborate on what we are doing here? The change seems to have been made only for childrensBooks and seriesBook.